go/vm/interpreter_mock.go

```diff
@@ -1,22 +1,5 @@
-//
-// Copyright (c) 2024 Fantom Foundation
-//
-// Use of this software is governed by the Business Source License included
-// in the LICENSE file and at fantom.foundation/bsl11.
-//
-// Change Date: 2028-4-16
-//
-// On the date above, in accordance with the Business Source License, use
-// of this software will be governed by the GNU Lesser General Public Licence v3
-//
-
```

Comment:
I guess it is not a big deal if we don't license generated code.
Is it there a make command to run the code generator?

Reply:
There is no make command, but a go command: `go generate ./...`

About the license ... we have not ported the license header script from Carmen yet (they have developed one we can hopefully reuse). I will fix the header manually for now, but we should port it at some point.

I also used a newer version of `gomock` to match the version used for generating the previous files. This reduces noise in the generated files.